### ISSUE TITLE
ValueError: Columns must be same length as key  when `len(date_cols) > 0`

### ISSUE THREAD

**Issue by eromoe** [AutoViML/featurewiz#47]
This error is from pandas 
throw at 
![image](https://user-images.githubusercontent.com/3938751/186895170-b5d1894c-1238-40e6-8645-33cd6d6150f8.png)

This happened when `len(date_cols)>0`  .

I found  `important_features` doesn't equal to `old_important_features` : 
```
In [4]: 'reg_date_hour' in important_features
Out[4]: True

In [5]: 'reg_date_hour' in old_important_features
Out[5]: False

In [6]: len(date_cols)
Out[6]: 2

In [14]: len(important_features) == len(old_important_features)
Out[14]: False

In [15]: len(important_features)
Out[15]: 460

In [16]: len(old_important_features)
Out[16]: 471


```

**Comment by eromoe** [comment]
Oh , you create some new feature for date col `reg_date_hour` , such as   
```
'reg_date_hour_hour', 
'reg_date_hour_dayofmonth',
```
saved in `old_important_features` , and try to update to `dataname[important_features]` which doesn't contain the new created features.

**Comment by AutoViML** [comment]
Hi @eromoe 👍 

You are working with a slightly older version. This problem has been found and fixed. Please upgrade to the latest version via:

```
!pip install featurewiz --no-deps --ignore-installed --upgrade
```
thanks
AutoVimal

**Comment by eromoe** [comment]
I installed from conda forge which fixed in `featurewiz  0.1.87` .
